### ISSUE TITLE
Sync read state

### ISSUE THREAD

**Issue by cutzenfriend** [lieuwex/whapp-irc#17]
Mark Messages as read somehow? So notifications on mobile devices will be removed.

**Comment by lieuwex** [comment]
Yeah I actually need to implement this, I didn't because I actually kinda like being able to respond to messages without WhatsApp showing I'm online.
But I understand it's nice to have, I will add it and add a flag to turn it off too.
